src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -451,6 +451,50 @@ private function isSimpleInstance($id, Definition $definition)
     }
 
     /**
+     * Checks if the definition is a trivial instance.
+     *
+     * @param Definition $definition
+     *
+     * @return bool
+     */
+    private function isTrivialInstance(Definition $definition)
```

Comment:
Instead of this method name (`isTrivialInstance()`) which feels like an internal detail, maybe we could use a more generic and future-proof name, such as `canBeInlined()` ?